packages/design-system/src/components/FilterChip/FilterChip.tsx

```diff
@@ -0,0 +1,75 @@
+import { ClearIcon } from '../ClearIcon';
+import React from 'react';
+import classNames from 'classnames';
+import uniqueId from 'lodash.uniqueid';
+
+const actionableKeys = ['Enter', 'Backspace', 'Delete'];
+
+export interface FilterChipProps {
+  /**
+   * Id for filter chip button. If not provided, a unique id will be automatically generated and used.
+   */
+  id?: string;
+  /**
+   * Class to be applied to the outer `<div>` that contains filter chip button.
+   */
+  className?: string;
+  /**
+   * Text for the filter chip
+   */
+  label: string;
+  /**
+   *  For screenreaders, text to read for removal
+   */
+  removeText: string;
```

Comment:
To stay consistent with other components (eg. Text field, Tooltip etc), can we change `removeText` prop name to `ariaLabel` instead

Reply:
Do you think that might be confusing that someone might think it's for the label itself instead of the close icon?

Reply:
Maybe something like `removeAriaLabel` ?

Reply:
Ah, looks like Autocomplete has `ariaClearLabel` - how have consistent format and use `ariaRemoveLabel` here?

Reply:
@nichia I actually think we could use `ariaClearLabel` since it's the label for the Clear Icon

Reply:
Changed here - https://github.com/CMSgov/design-system/pull/996/commits/994327d10654571e12637cb90a9294faae5de72e